src/olympia/devhub/templates/devhub/new-landing/components/publish_your_addon.html

```diff
@@ -11,12 +11,12 @@
       <h2 class="DevHub-content-header">{{ _('Publish Your Add-on') }}</h2>
       <p>
         {% trans %}
-        Have you an add-on ready? Submit your extension or theme to be
+        Have an add-on ready? Submit your extension or theme to be
```

Comment:
"Have you any add-ons, matey?!"

The old copy (clearly a typo) is like pirate talk ☠️⛵️